src/mutation.ts

```diff
@@ -14,7 +14,7 @@
 
 import arrify = require('arrify');
 import * as is from 'is';
-import * as Long from 'long';
+import Long = require('long');
```

Comment:
ah, this is probably why long needs to be a dependency. Do you know why the way its being imported is changing to require?

Reply:
IIRC, some compiler or linting error wasn't letting us do it the old way, but we must have made a recent change because it doesn't seem to be a problem anymore so I changed it back to an import.